scripts/release-notes/gen_ec_release_notes.py

```diff
@@ -110,6 +110,35 @@ def main():
     """
     global GITHUB_TOKEN
 
+    # The script runs as
+    # .../microshift/scripts/release-notes/gen_ec_release_notes.py
+    # and we want the path to
+    # .../microshift
+    root_dir = pathlib.Path(__file__).parent.parent.parent
+    version_makefile = root_dir / 'Makefile.version.aarch64.var'
+    # Makefile contains something like
+    #   OCP_VERSION := 4.16.0-0.nightly-arm64-2024-03-13-041907
+    # and we want
+    #   4.16
+    version = version_makefile.read_text('utf8').split('=')[-1].strip().split('.')[:2]
```

Comment:
Is the `version` a list? Looks like it due to `[:2]` and usage in line 136, but comment `4.16` suggests it's a `str`.
Do you think we could make some changes to have less list operations? Could we add a `struct {major, minor}` or at least `major` `minor` integers? All that `[idx]` (together with line 136) is a bit difficult to read

Reply:
Yeah, this could be written clearer. Maybe even as a regex. :-)

The comment is wrong here. `version` ends up a list, and then `versions_to_scan` below gets 2 strings.

I'll fix this up.